packages/codecs/ilp/test/parser.spec.ts

```diff
@@ -33,7 +33,8 @@ describe('Parser', function () {
         it(test.name, function () {
           const binary = Buffer.from(test.binary, 'hex')
 
-          const parsed = Parser.deserializeIlpPrepare(binary)
+          // TODO
+          const parsed: any = Parser.deserializeIlpPrepare(binary)
```

Comment:
These tests are breaking the types ... by setting data on `parsed` to a string where it's declared as a Buffer ... so `any` old fix will do ...